java/opendcs/src/main/java/decodes/tsdb/groupedit/TsListPanel.java

```diff
@@ -247,6 +242,211 @@ public void plot()
+                                int count = TsImporter.importTimeSeriesFile(theDb, filePath, tsIdsForThisFile,
+                                    tz, settings.siteNameTypePreference);
+                                totalCount += count;
+                                publish("  Successfully imported " + count + " time series from " + fileName);
+                            }
+                            catch (Exception ex)
+                            {
+                                publish("  Error processing file " + fileName + ": " + ex.getMessage());
+                                log.error("Error importing from file: " + fileName, ex);
+                            }
+                        }
+
+                        publish("Successfully imported " + totalCount + " time series total.");
+                        return totalCount;
+                    }
+
+                    @Override
+                    protected void process(java.util.List<String> chunks)
+                    {
+                        // Add progress messages to the dialog
+                        for (String message : chunks)
+                        {
+                            progressDlg.addToProgress(message);
+                        }
+                    }
+
+                    @Override
+                    protected void done()
+                    {
+                        try
+                        {
+                            Integer totalCount = get();
+
+                            // Open each imported TSID in the GUI
+                            for (String tsIdStr : allSelectedTsIds)
+                            {
+                                try
+                                {
+                                    // Get the TimeSeriesIdentifier from the database
+                                    try (opendcs.dai.TimeSeriesDAI timeSeriesDAO = theDb.makeTimeSeriesDAO())
+                                    {
+                                        TimeSeriesIdentifier tsid = timeSeriesDAO.getTimeSeriesIdentifier(tsIdStr);
+
+                                        // Check if this TSID is already being edited
+                                        if (!myFrame.makeEditPaneActive(tsid))
+                                        {
+                                            // Open new edit panel for this TSID
+                                            TsSpecEditPanel editPanel = new TsSpecEditPanel(myFrame);
+                                            editPanel.setTsSpec((CwmsTsId)tsid);
+                                            myFrame.addEditTab(editPanel, "" + tsid.getKey());
+                                        }
+                                    }
+                                }
+                                catch (Exception ex)
+                                {
+                                    log.warn("Could not open edit panel for TSID '{}': {}", tsIdStr, ex.getMessage());
```

Comment:
use the LoggingEventBuilder form here, we want more than just the basic exception message if there's an error here.